fd/example_test.go

```diff
@@ -0,0 +1,47 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package fd_test
+
+import (
+	"fmt"
+	"math"
+
+	"github.com/gonum/diff/fd"
+)
+
+func ExampleDerivative() {
+	f := func(x float64) float64 {
+		return math.Sin(x)
+	}
+	// Compute the first derivative of f at 0 using the default settings.
+	fmt.Println("f'(0) ≅", fd.Derivative(f, 0, nil))
```

Comment:
Can we use "≈" instead? I've seen ≅ represent a couple of concepts, even though the unicode standard agrees with you.